packages/core-components/src/components/Table/Table.tsx

```diff
@@ -470,6 +477,32 @@ export function Table<T extends object = {}>(props: TableProps<T>) {
   const columnCount = columns.length;
   const Body = useCallback(
     bodyProps => {
+      if (initialContentLoading) {
```

Comment:
I guess we will update this once the table changes are finalized right in the #19343